### DISCUSSION TITLE
Gnome Shell

### DISCUSSION THREAD

**Discussion by mambucodev** [catppuccin/catppuccin#2580] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

I'm not very good at making gnome-shell themes, as I've never seen one, but if anybody here knows how to make one I'm really forward to helping and contributing to this port!
I haven't found wikis or stuff, but we could essentially go into [other themes](https://www.gnome-look.org/browse?cat=134&ord=latest) and see what's going.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by 42willow** [comment]
Hi 👋

There was already a GTK/gnome shell theme here: https://github.com/catppuccin/gtk - unfortunately it was archived due to being terribly difficult to maintain (see https://github.com/catppuccin/gtk/issues/262)

catppuccin has been ported into the original upstream [colloid](https://github.com/vinceliuice/Colloid-gtk-theme) but in my opinion you are better off using the archived theme until it breaks as colloid has some issues.

Personally, I am using the default adwaita theme with a few overrides through stylix https://github.com/danth/stylix/blob/master/modules/gtk/gtk.mustache

you can achieve something similar with this https://github.com/olifloof/catppuccin-gtk-css

**Comment by 42willow** [comment]
Related #2511

**Comment by elisesouche** [comment]
I have made a script that changes the GNOME Shell's color palette to Catppuccin, generating a theme that can be used with User Theme. It simply patches the default GNOME CSS to change the colors, it is not a full blown theme like the old GTK port. But that means that it should be more stable, hopefully.

It is not really in the form of a port, and tbh I don't really have the motivation to make it comply with the rules for ports. But if someone wants to package it, they are welcome.

Link: https://github.com/elisesouche/gnome-catppuccin
